cibyl/orchestrator.py

```diff
@@ -192,6 +158,53 @@ def setup_sources(self):
                         if source.enabled:
                             source.setup()
 
+    def load_features(self):
+        """Read user-requested features and setup the right argument to query
+        the information for them."""
+        features_loader = FeaturesLoader()
+        user_features = self.parser.ci_args.get('features')
+        if not user_features:
+            return []
+        if user_features and not user_features.value:
+            # throw error in case cibyl is called with --features argument but
+            # without any specified feature
+            msg = "No feature specified. Choose one of the following "
+            msg += "features:"
+            msg += features_loader.get_string_all_features()
+            raise InvalidArgument(msg)
+        return [features_loader.get_feature(feature_name)
+                for feature_name in user_features.value]
+
+    def run_features(self, system, features_to_run):
+        """Run user-requested features."""
```

Comment:
Indicate that the output will be found on the 'system' argument.

Reply:
I'll expand the description here